lib/tbot/config/service_workload_identity_aws_ra.go

```diff
@@ -77,6 +77,19 @@ type WorkloadIdentityAWSRAService struct {
 	// defaults to 1 hour.
 	SessionRenewalInterval time.Duration `yaml:"session_renewal_interval"`
 
+	// CredentialProfileName is the name of the AWS credentials profile to
+	// write to. If unspecified, the profile will be named "default".
+	CredentialProfileName string `yaml:"credential_profile_name,omitempty"`
+
+	// ArtifactName is the name of the artifact to write to. This is the
+	// filename of the file that will be written to the destination. This is
+	// by default "aws_credentials".
+	ArtifactName string `yaml:"artifact_name,omitempty"`
+	// OverwriteCredentialFile is a flag that indicates whether the output
+	// should overwrite the existing credentials file rather than merging with
+	// it.
+	OverwriteCredentialFile bool `yaml:"overwrite_credential_file,omitempty"`
```

Comment:
I was thinking of perhaps using a string enum here rather than a bool - in case we wanted to add some stricter modes in future (e.g "don't merge and don't overwrite - a safe mode"). WDYT

Reply:
Sounds good to me